Examples/Evaluation/CNTKLibraryCSEvalCPUOnlyExamples/CNTKLibraryCSEvalExamples.cs

```diff
@@ -738,6 +738,68 @@ public static void EvaluationSingleSequenceUsingSparse(DeviceDescriptor device)
             }
         }
 
+        public static void ValueCopyToSparseCSCTest<T>(DeviceDescriptor device)
```

Comment:
Moved test code to memorytests.cs. Good to know the mechanism for packaging samples. I did not find models with sparse output. Will look into it and add samples once find one.